src/extension/tests/Test_ExtEnvHandler.py

```diff
@@ -233,7 +233,7 @@ def test_get_temp_folder_success(self):
         temp_folder_path = ext_env_handler.get_temp_folder()
 
         # validate path
-        self.assertEquals(temp_folder_path, ext_env_handler.temp_folder)
+        self.assertEqual(temp_folder_path, ext_env_handler.temp_folder)
```

Comment:
From this PR - https://github.com/Azure/LinuxPatchExtension/pull/306 
(Py 3.12 failure. That PR should ideally go in first.)